src/midiEditorCommands.cpp

```diff
@@ -212,10 +206,13 @@ void previewNotes(MediaItem_Take* take, const vector<MidiNote>& notes) {
 		(UINT)(minLength ? minLength * 1000 : DEFAULT_PREVIEW_LENGTH), previewDone);
 }
 
-void cancelMidiPreviewNotesOff() {
+void cancelMidiPreviewDoneTimer(bool notesOff) {
```

Comment:
This function name seems specific to the timer, but it also queues/sends note offs. Perhaps we could just call it cancelMidiPreview?